### PR TITLE
Kg/windows guide

### PR DESCRIPTION
**Pull request by kagarmoe**
### Description

Building on the nav-bar changes, this PR aggregates the Microsoft content from the docs in one place. Most of this content already existed but was in neither the TOC nor nav-bar. The plan is to merge this PR and then work through updates to the reference material in separate PRs.

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [x ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: chef_master/source/_templates/nav-docs.html [hunk 1]
```diff
@@ -126,8 +131,44 @@
           },
           {
             "title": "Microsoft Windows Guide",
-            "hasSubItems": false,
-            "url": "/windows.html"
+            "hasSubItems": true,
+            "subItems": [
+              {
+                "title": "Microsoft Azure",
```
Comment (tas50) [inline chef_master/source/_templates/nav-docs.html:106]: I'm not sure if we consider Azure to be a Windows things. Lots of people are running almost exclusively Linux workloads there.

File: chef_master/source/_templates/nav-docs.html [hunk 2]
```diff
@@ -126,8 +131,44 @@
...
+                "hasSubItems": false,
+                "url": "/install_windows.html"
+              },
+              {
+                "title": "Azure Powershell_Cmdlets",
+                "hasSubItems": false,
+                "url": "azure_powershell.html"
+              },
+              {
+                "title": "Knife Azure",
```
Comment (tas50) [inline chef_master/source/_templates/nav-docs.html:131]: Same as above

File: chef_master/source/_templates/nav-docs.html (hunk 1 above)
Comment (kagarmoe) [reply chef_master/source/_templates/nav-docs.html:106]: We could call it a "Chef Microsoft Guide"?

Comment (stuartpreston) [reply chef_master/source/_templates/nav-docs.html:106]: Maybe a "Chef on Windows" guide? As this seems to be Windows specific here.

Comment (kagarmoe) [reply chef_master/source/_templates/nav-docs.html:106]: Went with @stuartpreston's "Chef on Windows Guide". I don't have a better name

File: chef_master/source/_templates/nav-docs.html (hunk 2 above)
Comment (kagarmoe) [reply chef_master/source/_templates/nav-docs.html:131]: I don't have a better answer right now. Maybe it will come as I write.

**Review by mjingle — APPROVED** [review]
tenor-97550397

**Pull request merged**